[16:07:39] < glen> bug
[16:07:50] < glen> i can't add some low level user to authorized repliers list
[16:08:05] < glen> but when it replies, i can convert note and choose add to recipient list... no error produced
[16:08:12] < glen> altho it's not included into list eighter
[16:08:57] < lordrashmi> low level = reporter?
[16:09:03] < glen> yeah sth like that
[16:09:23] < glen> but funny is, if he has no account in eventum, i would be allowed to add his email address
[16:09:44] < glen> ie, i remember one issue where it worked earlier, that outside person was able to reply emails
[16:12:02] < lordrashmi> glen: include/class.authorized_replier.php, line 196
[16:12:21] < lordrashmi> change <= to ==
[16:12:42] < glen> will this be in next eventum too?
[16:12:49] < glen> or it will be my local fix only?
[16:13:18] < lordrashmi> committing now
[16:13:23] < glen> nice
[16:13:29] < lordrashmi> be glad being in prague makes me in a good mood

--- ./include/class.authorized_replier.php~	2005-04-22 18:34:16.000000000 +0300
+++ ./include/class.authorized_replier.php	2005-05-10 16:13:06.000000000 +0300
@@ -193,7 +193,7 @@
     function addUser($issue_id, $usr_id, $add_history = true)
     {
         // only users above customer role can be authorized repliers. All customers should already be able to reply.
-        if (User::getRoleByUser($usr_id, Issue::getProjectID($issue_id)) <= User::getRoleID("Customer")) {
+        if (User::getRoleByUser($usr_id, Issue::getProjectID($issue_id)) == User::getRoleID("Customer")) {
             return -2;
         }
         
